frontend/src/Temp.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useContext } from "react";
+import { UserContext } from "./UserContext.js";
+import { Link } from "react-router-dom";
+
+function Temp() {
```

Comment:
What is "Temp"? Could you give this file a more descriptive name? Again please make sure to give all files and variables descriptive names to make it easy for others working in the codebase to understand it, which speeds up project development